.changeset/six-cups-promise.md

```diff
@@ -0,0 +1,8 @@
+---
+'@commercetools-docs/gatsby-theme-api-docs': patch
```

Comment:
Since it's changing the configuration interface, isn't this strictly spoken a major release?

Reply:
Not really, the deprecated option is still there for backwards compatibility

Reply:
ok, I oversaw that.